build.gradle.kts

```diff
@@ -267,3 +247,19 @@ tasks.register("buildNdkIntegrationTestsArtifacts") {
 }
 
 nightlyTestsCoverageConfig(threshold = 0.9f)
+kover {
+    isDisabled = false
+    disabledProjects = setOf(
+        "instrumented",
+        "sample",
+        "tools",
+        "integration",
+        "nightly-tests",
+        "kotlin",
+        "detekt",
```

Comment:
Kover `disabledProjects` doesn't support full path to the project? Like `tools:detekt`.

Reply:
No unfortunately not.